python/grass/jupyter/__init__.py

```diff
@@ -13,23 +13,64 @@
 #            License (>=v2). Read the file COPYING that comes with GRASS
 #            for details.
 
-"""Display classes and setup functions for running GRASS GIS in Jupyter Notebooks
+"""The *grass.jupyter* is a convenient GRASS GIS interface for Jupyter notebooks.
+
+Python is a great tool for data science and scientific computing. Jupyter_ is an
+environment with computational notebooks which makes it even better for tool for
+analysis and workflow prototyping. Computational notebooks are documents combining
+code, text, and results such as figures and tables. JupyterLab is an environment where
+you interact with all these parts. You can install it locally on your machine or
+use it online from some service provider.
 
 The *grass.jupyter* subpackage improves the integration of GRASS GIS and Jupyter
-Notebooks. The original version was written as part of Google Summer of Code in 2021
-and experimental version was included in GRASS GIS 8.0. Since then, much more
-development happened adding better session handling and rendering of additional data
-types.
+notebooks comparing to the standard Python API. The original version was written
+as part of Google Summer of Code in 2021 and experimental version was included in
+GRASS GIS 8.0. Since then, much more development happened adding better session
+handling and rendering of additional data types.
 
-For standard usage, simply import the top level package with a convenient alias, e.g.,::
+Usage
+=====
 
->>> import grass.jupyter as gj
+To start using it in a notebook, import the top level package with a convenient alias,
+such as *gj*, like this::
 
-The objects in submodules and names of submodules may change in the future.
+>>> import grass.jupyter as gj
 
 .. note::
     To import the package, you need to tell Python where the GRASS GIS Python package
-    is. Please, refer to example notebooks for an example of the full workflow.
+    is unless you manually set this on your system or in command line. Please, refer
+    to example notebooks linked below for an example of the full workflow.
+
+.. note::
+    On Windows, there is no system Python and GRASS GIS needs to use its own Python.
+    Jupyter needs to be installed into that Python. Please, refer to the wiki_
```

Comment:
Is it possible to add a link to that wiki? Otherwise how will users know which wiki to search for

Reply:
The wiki is linked:

![Screenshot from 2022-04-18 17-55-39](https://user-images.githubusercontent.com/5449060/163883932-729578c1-5c63-44cb-a8ce-035c14fcd637.png)

Reply:
I see, so the `_` there means that's a link! Sorry for the noise :woman_facepalming: I am not familiar with this syntax

Reply:
At this point, the syntax (reStructuredText) is very Python-specific, quite wide spread in Python projects, but rare otherwise. Some addition to Sphinx for docstrings to be Markdown is desired but [seems unavailable](https://github.com/executablebooks/MyST-Parser/issues/228).